doc/modules/cassandra/pages/getting-started/mtlsauthenticators.adoc

```diff
@@ -0,0 +1,133 @@
+= Getting started with mTLS authenticators
+
+When a certificate based authentication protocol like TLS is used for client and
+Internode connections, `MutualTlsAuthenticator` & `MutualTlsInternodeAuthenticator`
+can be used for the authentication by leveraging the client certificates from the
+SSL handshake.
+
+After SSL handshake, identity from the client certificates is extracted and only
+authorized users will be granted access.
+
+== What is an Identity
+
+One can define their own identity for certificates by extracting some fields or
+information from the certificates. one can implement the interface `MutualTlsCertificateValidator`
+to provide logic for validating & extracting identity from the certificates.
+
+There is a default implementation of `MutualTlsCertificateValidator` with
+https://spiffe.io/docs/latest/spiffe-about/spiffe-concepts/[SPIFFE] as the identity
+of the certificates.This requires spiffe to be present in the SAN of the certificate.
+One can implement their own validator that suits their needs.
+
+For example, instead of using `SPIFFE` based validator a `CN` based validator can be
+implemented.
```

Comment:
```suggestion
Instead of using `SPIFFE` based validator a `CN` based validator could by an operator-supplied custom class that implements `MutualTlsCertificateValidator`.
```